### ISSUE TITLE
Getting warning on button: bound output click does not exist

### ISSUE THREAD

**Issue by chalin** [dart-lang/angular_analyzer_plugin#591]
## Environment

- Angular 5.0.0-beta+1
- Dart SDK 2-dev.69
- WebStorm 2018.2 RC

## Steps to reproduce

1. Download the [dependency-injection](https://github.com/angular-examples/dependency-injection) example app.
2. Open `dependency-injection` in WebStorm.
3. Get the app's dependencies (pub get).

## What happens?

The following warning is displayed:

```
warning: The bound output click does not exist on any directives or on the element (nonexist_output_bound at [dependency_injection] lib/app_component.dart:20)
```

The warning is referring to the `(click)` binding in this template:

```dart
@Component(
  selector: 'my-app',
  template: '''
    ...
      <button (click)="nextUser()">Next User</button>
    ...
  ''',
  // ...
)
class AppComponent {
  // ...
  void nextUser() { ... }
  // ...
}
```

I don't recall seeing this warning before. Anyhow, a button certainly emits click events.

cc @kwalrath
